src/sidebar/components/Excerpt.js

```diff
@@ -20,20 +20,32 @@ import { applyTheme } from '../helpers/theme';
  * @param {InlineControlsProps} props
  */
 function InlineControls({ isCollapsed, setCollapsed, linkStyle = {} }) {
-  const toggleLabel = isCollapsed ? 'More' : 'Less';
-
   return (
-    <div className="Excerpt__inline-controls">
-      <div className="Excerpt__toggle-container">
+    <div
+      className={classnames(
+        // Position these controls at the bottom right of the excerpt
+        'absolute block right-0 bottom-0',
+        // Give extra width for larger tap target and gradient fade
+        // Fade transparent-to-white left-to-right to make the toggle
+        // control text (More/Less) more readable above other text.
+        // This gradient is implemented to-left to take advantage of Tailwind's
+        // automatic to-transparent calculation: this avoids Safari's problem
+        // with transparents in gradients:
+        // https://bugs.webkit.org/show_bug.cgi?id=150940
```

Comment:
This Safari bug was fixed on 2021-12-23, which I think means that the fix would have shipped in Safari 15.3. We'll still need the workaround for a couple of years though.